src/utils/helpers.ts

```diff
@@ -72,3 +86,20 @@ export const decode = (string: string) =>
     Buffer.from(string, CONTENT_ENCODINGS.base64 as BufferEncoding).toString(CONTENT_ENCODINGS.utf8 as BufferEncoding);
 export const encode = (string: string) =>
     Buffer.from(string, CONTENT_ENCODINGS.utf8 as BufferEncoding).toString(CONTENT_ENCODINGS.base64 as BufferEncoding);
+
+const isLegacyAppManifest = (decodedAppManifestContent: any) => decodedAppManifestContent instanceof Array;
+
+export const updateAppManifestContent = (decodedAppManifestContent: any, appName: string, vspecPath: string, dataPoints: any[]) => {
```

Comment:
I'd argue this is not really suited for the helpers module. Maybe move it back to the project generator?